src/ray/raylet/node_manager.cc

```diff
@@ -237,22 +237,23 @@ ray::Status NodeManager::RegisterGcs() {
 }
 
 void NodeManager::KillWorker(std::shared_ptr<Worker> worker) {
+#ifdef _WIN32
+  // TODO(mehrdadn): Implement implement graceful process termination mechanism
+#else
```

Comment:
We should define this macro condition in the `process.h` only. 
```c++
worker->Process()->Kill(SIGTERM);
or 
worker->KillProcess(SIGTERM);

```

Reply:
Sorry I'm confused which macro condition you're referring to (and why should it be in the header?), but I think you started reviewing this PR a little early -- it's still a work in progress :) sorry for the confusion.